source/query-bar.txt

```diff
@@ -164,8 +184,37 @@ sort by ``name`` in ascending order.
 
    { year: -1, name: 1 }
 
-For more information on sort, refer to the MongoDB Manual :manual:`sort
-</reference/method/cursor.sort>`.
+For more information on sort, refer to the
+:manual:`MongoDB Manual </reference/method/cursor.sort>`.
+
+.. _query-bar-collation:
+
+Collation
+~~~~~~~~~
+
+:manual:`Collation </reference/collation/>` allows you to specify
+language-specific rules for string comparison, such as rules for
+lettercase and accent marks. Use this query bar option to specify
+collation for the current query.
```

Comment:
The second sentence feels like filler to me.  Doesn't really add anything to the first sentence. Perhaps merge with the first sentence? "Use the Collation query bar option to ..."

Reply:
Agreed. Done.